mm2src/coins/utxo/rpc_clients.rs

```diff
@@ -2709,36 +2726,35 @@ fn electrum_connect(
     }
 }
 
+// electrum_request should always return JsonRpcErrorType::Transport Error
```

Comment:
Firs of all, could you please make this a document comment? Structure names should be appropriately highlighted.
```rust
/// `electrum_request` should always return [`JsonRpcErrorType::Transport`] error.
```

Secondly, I think `electrum_request` can return another error kind.
This limitation leads to the fact that the function takes one more argument.

Reply:
done